eternalcore-paper/src/main/java/com/eternalcode/containers/AdditionalContainerType.java

```diff
@@ -1,10 +1,58 @@
 package com.eternalcode.containers;
 
-public enum AdditionalContainerType {
-    ANVIL,
-    STONE_CUTTER,
-    CARTOGRAPHY_TABLE,
-    GRINDSTONE,
-    LOOM,
-    SMITHING_TABLE,
+
+import org.bukkit.entity.Player;
+
+import java.util.Objects;
+import java.util.function.Consumer;
+
+public class AdditionalContainerType {
+
+    public static final AdditionalContainerType ANVIL = new AdditionalContainerType("anvil", player -> player.openAnvil(null, true));
+    public static final AdditionalContainerType STONE_CUTTER = new AdditionalContainerType("stone_cutter", player -> player.openStonecutter(null, true));
+    public static final AdditionalContainerType GRINDSTONE = new AdditionalContainerType("grindstone", player -> player.openGrindstone(null, true));
+    public static final AdditionalContainerType CARTOGRAPHY_TABLE = new AdditionalContainerType("cartography_table", player -> player.openCartographyTable(null, true));
+    public static final AdditionalContainerType LOOM = new AdditionalContainerType("loom", player -> player.openLoom(null, true));
+    public static final AdditionalContainerType SMITHING_TABLE = new AdditionalContainerType("smithing_table", player -> player.openSmithingTable(null, true));
+    public static final AdditionalContainerType ENCANTING_TABLE = new AdditionalContainerType("enchanting_table", player -> player.openEnchanting(null, true));
```

Comment:
To nie zadziała tak samo jak w innych przypadkach, nie możesz sforsować otwarcie enchantingu jeżeli w pobliżu nie ma bloku bo silnik musi przekalkulować biblioteczki

Reply:
Jak nie ustawisz force na true to nie otworzy ci enchantu, nie ma tutaj chyba dobrego rozwiązania.

Reply:
@failutee W sumie to dzięki, nie wiedziałem, że tak to działa, myślę że właśnie będzie trzeba to trochę oszukać